app/code/Magento/MediaGalleryApi/Api/Data/AssetInterface.php

```diff
@@ -13,6 +13,7 @@
 /**
  * Represents a media gallery asset which contains information about a media asset entity such
  * as path to the media storage, media asset title and its content type, etc.
+ * @api
```

Comment:
What is "source"? (in `getSource()`). As it's an arbitrary string, it's not very clear what can it be.

Reply:
Improved method comment